pywbem/_listener.py

```diff
@@ -762,31 +781,14 @@ def logger(self):
         """
         :class:`py:logging.Logger`: Logger object for this listener.
 
-        Each listener object has its own separate logger object that is
-        created via :func:`py:logging.getLogger`.
+        Each listener object has its own separate logger object with the name:
 
-        The name of this logger object is:
+          `'pywbem.listener.{id}'`
 
-          ``pywbem.listener.{id}``
```

Comment:
Did not mean to start a conversation just single review comments

Reply:
Ok